storybookjs/storybook
# Themes: Run postinstall in shell for windows

## What I did
- [x] Run postinstall script in shell for windows
- [x] Break up command string into command and args

## Checklist for Contributors

### Testing

#### Manual testing

1. Use pnpm
2. Create a vue-cli sandbox
3. `pnpm add -D tailwindcss postcss autoprefixer`
4. `pnpm dlx tailwindcss init -p`
5. Add tailwind global css file with the following code
   ```css
   @tailwind base;
   @tailwind components;
   @tailwind utilities;
   ```
6. `pnpm dlx storybook@latest add @storybook/addon-themes@<CANARY_VERSION>`
7. CLI should not error out

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release


This pull request has been released as version [`0.0.0-pr-24389-sha-88c59837`](https://npmjs.com/package/@storybook/cli/v/0.0.0-pr-24389-sha-88c59837). Install it by pinning all your Storybook dependencies to that version.
<details>
<summary>More information</summary>

| | |
| --- | --- |
| **Published version** | [`0.0.0-pr-24389-sha-88c59837`](https://npmjs.com/package/@storybook/cli/v/0.0.0-pr-24389-sha-88c59837) |
| **Triggered by** | @cdedreuille |
| **Repository** | [storybookjs/storybook](https://github.com/storybookjs/storybook) |
| **Branch** | [`postinstall/themes`](https://github.com/storybookjs/storybook/tree/postinstall/themes) |
| **Commit** | [`88c59837`](https://github.com/storybookjs/storybook/commit/88c59837993dae7b472324a7027fb5815ebdc529) |
| **Datetime** | Thu Oct  5 14:12:46 UTC 2023 (`1696515166`) |
| **Workflow run** | [6420253530](https://github.com/storybookjs/storybook/actions/runs/6420253530) |

To request a new release of this pull request, mention the `@storybookjs/core` team.

_core team members can create a new canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=24389`_
</details>
